api/AbortPaymentEvent.json

```diff
@@ -0,0 +1,191 @@
+{
+  "api": {
+    "PaymentRequestEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AbortPaymentEvent",
+        "support": {
+          "chrome": {
+            "version_added": "61",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "Service Worker payment apps",
+                "value_to_set": "Enabled"
+              }
+            ]
+          },
+          "chrome_android": {
+            "version_added": "61",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "Service Worker payment apps",
+                "value_to_set": "Enabled"
+              }
+            ]
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
```

Comment:
Since w3c/payment-handler#207 hasn’t been merged yet, this should probably be `false`.